resources/elbv2-listenerrule.go

```diff
@@ -0,0 +1,141 @@
+package resources
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/elbv2"
+	"github.com/rebuy-de/aws-nuke/v2/pkg/types"
+	"github.com/sirupsen/logrus"
+)
+
+var elbv2ListenerRulePageSize int64 = 400 // AWS has a limit of 100 rules per listener
+
+type ELBv2ListenerRule struct {
+	svc      *elbv2.ELBV2
+	elb      *elbv2.LoadBalancer
+	listener *elbv2.Listener
+	rule     *elbv2.Rule
```

Comment:
Do we really need to store whole objects here? Looks like we're only using ARNs and names from the resources.

Reply:
It's indeed not required, I've changed to only store what's necessary.